src/etc/rc

```diff
@@ -73,9 +73,7 @@ while read FS_PART FS_MNT FS_TYPE FS_MORE; do
 		FS_DEV=$(glabel status -as | grep ${FS_DEV} | awk 'match($3, /^[a-z]+[0-9]+/) { print substr( $3, RSTART, RLENGTH )}')
 		if [ "${FS_MNT}" = "/" -a -f ${GROWFS_MARKER} ]; then
 			# hammertime!
-			gpart recover ${FS_DEV}
-			gpart resize -i 1 ${FS_DEV}
-			growfs -y ${FS_MNT}
+			/etc/rc.d/growfs onestart
```

Comment:
woks for me and also adds ZFS support... yet.. if we go through all the trouble now to do a one-shot script without arguments wouldn't it be better to move it below the fstab loop? just need to check for GROWFS_MARKER this way. :)

Reply:
Do we have any benefits, moving it outside the loop?
if we let it within the loop, the resize still happens offline, or am i wrong?

I would still prefer to do partition changes with not mounted filesystems ;)

Reply:
the point is to remove obsolete and unused code

the loop here only reads fstab and configures before mounting the file systems

the next loop will mount the file system

simple solution: use the space in between to execute the growfs code :)

Reply:
Well, i changed it now, will update the pr when tests are finished.

Is the whole fstab loop (tunefs, trim and so on) still needed?

Reply:
yes, we need that

Reply:
files are updated and first test worked without problems :)

Reply:
RC will be a good opportunity to test this later. Should also work fine on 11.2 from was I can tell. Nice stuff this.